nyc_data/ppe/aggregations.py

```diff
@@ -40,8 +40,8 @@ def asset_rollup(
         rollup_fn: Callable[[dc.Item], any] = lambda x: x,
         estimate_demand=True
 ) -> Dict[str, AssetRollup]:
-    relevant_deliveries = Delivery.objects.prefetch_related("purchase").filter(
-        delivery_date__gte=time_start, delivery_date__lte=time_end, replaced=False
+    relevant_deliveries = Delivery.objects.prefetch_related("purchase", "source").filter(
```

Comment:
I'm curious - what does "source" relate to here? I would have thought it would be a column in Delivery, but that doesn't seem to be the case (https://github.com/RTCovid/nyc-ppe/blob/track-imports/nyc_data/ppe/models.py#L75)

Reply:
Bah, think I answered my own question. Didn't realize `Delivery` inherited BaseModel - https://github.com/RTCovid/nyc-ppe/blob/track-imports/nyc_data/ppe/models.py#L46